packager/app/packager_main.cc

```diff
@@ -334,7 +334,7 @@ base::Optional<PackagingParams> GetPackagingParams() {
     // the smallest unit of video. Therefore, in low latency mode,
     // each fragment will only contain one chunk, defaulting the
     // fragment duration to the shortest time possible
-    // and making a user specified --framgnet_dration irrelevant.
+    // and making a user specified --fragment_dration irrelevant.
```

Comment:
Still missing a "u" in "duration"

Reply:
Fixed! :) https://github.com/CaitlinOCallaghan/shaka-packager/pull/3/files#r681359831